etna/transforms/math/binary_operator.py

```diff
@@ -0,0 +1,181 @@
+from enum import Enum
+from typing import List
+from typing import Optional
+
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.transforms.base import ReversibleTransform
+
+
+class BinaryOperator(str, Enum):
+    """Enum for mathematical operators from pandas."""
+
+    add = "+"
+    sub = "-"
+    mul = "*"
+    div = "/"
+    floordiv = "/="
```

Comment:
Could someone explain what it does? Shouldn't it be `//`?

Reply:
It's literal for floordiv, i checked that it works correctly

Reply:
As I understand "/=" means divide and assign, not floor div. You can't write `2 /= 3` in python, but you can write `2 // 3`.